doc/sphinx/addendum/type-classes.rst

```diff
@@ -446,26 +438,47 @@ few other commands related to typeclasses.
    + When considering local hypotheses, we use the union of all the modes
      declared in the given databases.
 
-   .. tacv:: typeclasses eauto @natural
-
-      .. warning::
-         The semantics for the limit :n:`@natural`
-         is different than for auto. By default, if no limit is given, the
-         search is unbounded. Contrary to :tacn:`auto`, introduction steps are
-         counted, which might result in larger limits being necessary when
-         searching with ``typeclasses eauto`` than with :tacn:`auto`.
+   + Note that this tactic is distinct from the similarly-named command
+     :cmd:`Typeclasses eauto`.
```

Comment:
`Typeclasses eauto` is used to parameterize `typeclass eauto`. I'm not sure that this bullet-point is making things better, especially given that the other bullet-points are about differences between `typeclass eauto` and `eauto`.